Core/src/org/sleuthkit/autopsy/modules/photoreccarver/PhotoRecCarverFileIngestModule.java

```diff
@@ -435,17 +441,32 @@ synchronized Path createModuleOutputDirectoryForCase() throws IngestModule.Inges
      *
      * @throws IngestModuleException
      */
-    public static File locateExecutable(String executableToFindName) throws IngestModule.IngestModuleException {
-        // Must be running under a Windows operating system.
-        if (!PlatformUtil.isWindowsOS()) {
-            throw new IngestModule.IngestModuleException(Bundle.unsupportedOS_message());
+    public static File locateExecutable() throws IngestModule.IngestModuleException, IOException {
```

Comment:
What is throwing IOException?  We should probably catch that in this method and deal with it appropriately.